nav2_bringup/README.md

```diff
@@ -0,0 +1,65 @@
+# Bringup
+
+## Before starting:
+- Build ROS2 from master branch
+- If working with Ubuntu 16, install [Gazebo 9](http://gazebosim.org/tutorials?tut=install_ubuntu&cat=install)
+  - In step #3:
```

Comment:
It was immediately obvious that the text was referring to step 3 n the gazebo install tutorial. Could make this explicit: "In step #3 in the gazebo installation instructions" or something like that.